### PR TITLE
sample for WTC

### PR DESCRIPTION
**Pull request by CarolynRountree**
Documentation for a model sample using WTC

### CODE REVIEW

File: documentation/1.9/content/samples/wtc-model.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+title: "Modeling a WTC configuration"
+date: 2021-11-01T10:19:24-05:00
+draft: false
+weight: 3
+description: "A domain model with a typical queue based configuration for a WebLogic Tuxedo Connector."
```
Comment (rosemarymarano) [inline documentation/1.9/content/samples/wtc-model.md:6]: queue based -> queue-based

File: documentation/1.9/content/samples/wtc-model.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+           AccessPoint: RemoteAccessPoint
+           AccessPointId: domain1
+         RemoteAccessPoint2:
+           LocalAccessPoint: LocalAccessPoint2
+           NWAddr: '//access-host:2500'
+           AccessPoint: RemoteAccessPoint2
+           AccessPointId: domain1
+
+```
+This sample shows a start-up class that is collected into the archive file under wlsdeploy/domainLibs. Refer to [Archive structure]({{< relref "/concepts/archive#archive-structure" >}}) .
```
Comment (rosemarymarano) [inline documentation/1.9/content/samples/wtc-model.md:74]: start-up -> startup
wlsdeploy/domainLibs -> `wlsdeploy/domainLibs` (code font)

File: documentation/1.9/content/samples/wtc-model.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+In this configuration, the WTC is queue based with remote and local access points. To see other WTC configuration options, use the model help as shown below.
```
Comment (rosemarymarano) [inline documentation/1.9/content/samples/wtc-model.md:75]: as shown below -> as shown in the following:

**Review by rosemarymarano — COMMENTED** [review]
Minor edits

**Review by rakillen — APPROVED** [review]

File: documentation/1.9/content/samples/wtc-model.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+   StartupClass:
+     WtcServerSideHelper:
+       Target: admin
+       ClassName: wlstest.functional.core.wtc.conversation.common.apps.Helper.WtcServerSideHelperImpl
+   WTCServer:
+     myWTCServer:
+       Target: admin
+       WTCExport:
+         'WTCExportedService-1':
+           ResourceName: QaWls2Conv2
```
Comment (ddsharpe) [inline documentation/1.9/content/samples/wtc-model.md:22]: Please call out for the reader what each of these fields maps to.  The names are not descriptive enough to be self-explanatory.

**Review by ddsharpe — CHANGES_REQUESTED** [review]

File: documentation/1.9/content/samples/wtc-model.md (hunk 4 above)
Comment (rakillen) [reply documentation/1.9/content/samples/wtc-model.md:22]: Maybe as # comments?

File: documentation/1.9/content/samples/wtc-model.md (hunk 2 above)
Comment (CarolynRountree) [reply documentation/1.9/content/samples/wtc-model.md]: how do you do code fonts? Missed this and still needs to be fixed.

Comment (rosemarymarano) [reply documentation/1.9/content/samples/wtc-model.md]: You apply a backward tick before and after the text; it looks like this `

**Review by rosemarymarano — APPROVED** [review]

**Review by ddsharpe — APPROVED** [review]

**Pull request merged**